deploy/README.md

```diff
@@ -190,7 +190,7 @@ or
 Arguments | Required | Type | Sample value
 ----------|-----------|-------|---------------
 environmentCode | yes | string | aoi
-pipelineName | yes | string | Allowed Values: custom-vision-model, custom-vision-model-v2
+pipelineName | no | string | Allowed Value: custom-vision-model
```

Comment:
This should be `yes` as well

Reply:
package.sh has a default value. so it is not a required parameter

Reply:
I was comparing the code that is in the main branch but didn't recognise that the default value was added on this PR.